app/ts/components/Navbar.tsx

```diff
@@ -15,11 +15,12 @@ export const Navbar = ({
 }) => {
 	const switchNetwork = async (e: Event) => {
 		const elm = e.target as HTMLSelectElement
-		const relayEndpoint = elm.value
+		const simulationRelayEndpoint = elm.value
+		const submitRelayEndpoint = elm.value
```

Comment:
Nitpick: `s/submit/submission` to match `simulation`.  (same comment throughout)